.github/workflows/continuous-integration.yaml

```
@@ -56,3 +56,10 @@ jobs:
               uses: codecov/codecov-action@v4
               with:
                   token: ${{ secrets.CODECOV_TOKEN }}
+
+    apiDocumentation:
+      runs-on: ubuntu-latest
```

yamllint

[warning] wrong indentation: expected 8 but found 6

(indentation)